common/lib/xmodule/xmodule/x_module.py

```diff
@@ -578,7 +581,21 @@ def bind_for_student(self, xmodule_runtime, field_data, user_id):
 
         # Set the new xmodule_runtime and field_data (which are user-specific)
         self.xmodule_runtime = xmodule_runtime
-        self._field_data = field_data
+
+        # if the descriptor doesn't have this attribute yet,
+        # set it here before we start wrapping it
+        if not hasattr(self, '_unwrapped_field_data'):
+            self._unwrapped_field_data = self._field_data
```

Comment:
Optional: You could also implement this as a `@lazy` property that returns `self._field_data`.